src/web.cpp

```diff
@@ -18,13 +18,15 @@
 #include "ratgdo.h"
 #include "comms.h"
 
+#include "EspSaveCrash.h"
 #include <arduino_homekit_server.h>
 #include <ESP8266WebServer.h>
 #include <ESP8266HTTPUpdateServer.h>
 #include <Ticker.h>
 #include "log.h"
 #include "utilities.h"
 
+EspSaveCrash saveCrash(1408, 0x400);
```

Comment:
I think you need to add the 3rd parameter "true".  I missed this on my first revision.